src/App.vue

```diff
@@ -7,10 +7,13 @@
           <img src="/images/logo.svg" alt="jamstack deploy" />
         </router-link>
 
-        <h2 class="text-base text-center text-black mb-10">
+        <h2 class="text-base text-center text-black">
           Measure cloud providers and check which fits better for your Jamstack
           App.
         </h2>
+        <a v-if="isHome" href="/about" class="mb-10 text-red text-sm"
```

Comment:
Can we replace `<a>` with `<router-link>`. This will allow us to keep SPA navigation

```js
<router-link to="/about">...</router-link>
```

Reply:
Way better!